di-ipv-evcs-stub/openAPI/evcs-external.yaml

```diff
@@ -115,19 +112,43 @@ paths:
           required: true
           schema:
             type: string
+            format: uuid
+            example: urn:uuid:d1823066-2137-4380-b0ba-4b61947e08e6
+          description: A valid uuid as user Id.
+        - name: state
+          in: query
+          required: false
+          schema:
+            $ref: "#/components/schemas/VCStates"
+            example: CURRENT, PENDING_RETURN or ALL
+          description: A valid comma separated state value or ALL to retrive VCs in all available states.
+        - name: afterKey
+          in: query
+          required: false
+          schema:
+            $ref: "#/components/schemas/VCSignature"
+            example: qf0yp7B1an7cEwBui7GFCF9NNCJhHxTZuMSh5ehZPmZ4J527okK3pRgdSpWX8DlBFiZS-rXA496egfcfI-neGQ
+          description: A VC signature to retrive next set of VCs, its used for pagination.
       responses:
         200:
           description: "The list of Verifiable Credentials"
           content:
             application/json:
               schema:
-                type: "object"
+                type: object
+                properties:
+                  vcs:
+                    type: string
+                    description: The SQS message ID.
+                    example: "bd8359d9-d559-47dd-9467-2a31e88a9e2d"
```

Comment:
Thing this should be type array, not an SQS message ID

Reply:
yeah missed it.